### PR TITLE
Added deprecating IE support topic

### PR DESCRIPTION
**Pull request by jm-trd-ms**

### CODE REVIEW

**Review by dan-wesley — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
> Whenever a user of IE visits an incompatible public site, the user **may** SUGGEST **might**  
The term **might** should be avoided.

> The new capability automatically **redirect** SB **redirects**  
Updated.  

```md
The new capability automatically redirects the user.
```  

> When **a user navigates using IE to a site that is incompatible** with IE, SUGGEST **an IE user navigates to a site that's incompatible**  
Updated.  

```md
When an IE user navigates to a site that is incompatible with IE, Windows automatically redirects the user to Microsoft Edge.
```

**Comment by dan-wesley** [comment]
Suggest you replace “may” with “will” then – may refers to permission to do something.

/**
* Dan Wesley (Experis)
*/

**Comment by jm-trd-ms** [comment]
> Suggest you replace “may” with “will” then – may refers to permission to do something.  

Thank you for the suggestion.  
Correct `may`, `should`, or `must` should be used for imperative actions.  
Use `may` for possible or optional actions, `should` for likely or recommended actions, and `must` for only or required actions.  
`will` should be avoided since it does not indicate a present tense action.

File: microsoft-edge/web-platform/deprecating-ie-support.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Many modern websites have designs that are incompatible with Internet Explorer \(IE\).  Whenever a user of IE visits an incompatible public site, the user may get a message stating the site is incompatible with the browser.  After the message is displayed, the user must manually switch to a different browser.  To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability.  The new capability automatically redirects the user.  When an IE user navigates to a site that is incompatible with IE, Windows automatically redirects the user to Microsoft Edge.  To review the sites on the list, navigate to [Need Microsoft Edge list][MicrosoftEdgeNeededgeV1].
+
+This article describes the following concepts.  
+
+*   The user experience for redirection  
+*   How a site gets added to the IE compatibility list  
+*   How to get your site added or removed from the list  
+    
+## Why is a site added to the CV List?
```
Comment (HarneetSidhana) [inline microsoft-edge/web-platform/deprecating-ie-support.md:20]: This should be "IE compatibility" list

File: microsoft-edge/web-platform/deprecating-ie-support.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+This article describes the following concepts.  
+
+*   The user experience for redirection  
+*   How a site gets added to the IE compatibility list  
+*   How to get your site added or removed from the list  
+    
+## Why is a site added to the CV List?  
+
+The Compatibility View List only adds a site when the following actions occur.
```
Comment (HarneetSidhana) [inline microsoft-edge/web-platform/deprecating-ie-support.md:22]: This should be "IE compatibility" list

File: microsoft-edge/web-platform/deprecating-ie-support.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+title: Moving users to Microsoft Edge from Internet Explorer
+author: MSEdgeTeam
+ms.date: 11/03/2020
+ms.author: msedgedevrel
+ms.prod: microsoft-edge
+keywords: microsoft edge, compatibility, web platform, internet explorer
+---
+# Moving users to Microsoft Edge from Internet Explorer 
+
+Many modern websites have designs that are incompatible with Internet Explorer \(IE\).  Whenever a user of IE visits an incompatible public site, the user may get a message stating the site is incompatible with the browser.  After the message is displayed, the user must manually switch to a different browser.  To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability.  The new capability automatically redirects the user.  When an IE user navigates to a site that is incompatible with IE, Windows automatically redirects the user to Microsoft Edge.  To review the sites on the list, navigate to [Need Microsoft Edge list][MicrosoftEdgeNeededgeV1].
```
Comment (HarneetSidhana) [inline microsoft-edge/web-platform/deprecating-ie-support.md:12]: Can we say, 

After the message is displayed, the user is expected to manually switch to a modern browser.

File: microsoft-edge/web-platform/deprecating-ie-support.md (hunk 1 above)
Comment (jm-trd-ms) [reply microsoft-edge/web-platform/deprecating-ie-support.md:20]: Updated.

```md
## Why is a site added to the IE compatibility?  
```

File: microsoft-edge/web-platform/deprecating-ie-support.md (hunk 3 above)
Comment (HarneetSidhana) [inline microsoft-edge/web-platform/deprecating-ie-support.md:12]: restate to 

To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability which automatically redirects users.

File: microsoft-edge/web-platform/deprecating-ie-support.md (hunk 2 above)
Comment (jm-trd-ms) [reply microsoft-edge/web-platform/deprecating-ie-support.md:22]: Updated.

```md
The IE compatibility List only adds a site when the following actions occur.  
```

File: microsoft-edge/web-platform/deprecating-ie-support.md (hunk 3 above)
Comment (jm-trd-ms) [reply microsoft-edge/web-platform/deprecating-ie-support.md:12]: Updated.

```md
After the message is displayed, the user is expected to manually switch to a modern browser.
```

Comment (jm-trd-ms) [reply microsoft-edge/web-platform/deprecating-ie-support.md:12]: Updated.

```md
To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability that automatically redirects users.
```

**Review by HarneetSidhana — APPROVED** [review]
LGTM

**Review by cew6992 — APPROVED** [review]

File: microsoft-edge/web-platform/deprecating-ie-support.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/web-platform/deprecating-ie-support.md]:
```md
To review the websites on the list, navigate to [Need Microsoft Edge list][MicrosoftEdgeNeededgeV1].
```  

```md
[MicrosoftEdgeNeededgeV1]: https://edge.microsoft.com/neededge/v1 "Need Microsoft Edge list v1 xml | Microsoft Edge"  
```

File: microsoft-edge/web-platform/deprecating-ie-support.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+The Compatibility View List only adds a site when the following actions occur.  
+
+*   Shows an IE user a message suggesting the user should use a different browser for compatibility reasons.  
+*   Owner requests to add the site to the IE compatibility list.  
+    
+## Update the IE Compatibility List  
+
+The IE compatibility list is an XML file on [microsoft.com][MicrosoftOfficialHome].  The list is regularly updated in response to user's and site developers requests to have sites added or removed.  Updates to the list are automatically downloaded onto user's machine.  
+
+Email the following information to [ietioedge@microsoft.com][MailtoMicrosoftIetioedge] for your site to be added or removed from the IE compatibility list:
```
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/web-platform/deprecating-ie-support.md]:
```md
Email the following information to [ietoedge@microsoft.com][MailtoMicrosoftIetoedge] for your website to be added or removed from the IE compatibility list.
```  

```md
[MailtoMicrosoftIetoedge]: mailto:ietoedge@microsoft.com "Send an email to ietoedge@microsoft.com"  
```

File: microsoft-edge/web-platform/deprecating-ie-support.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+## Update the IE Compatibility List  
+
+The IE compatibility list is an XML file on [microsoft.com][MicrosoftOfficialHome].  The list is regularly updated in response to user's and site developers requests to have sites added or removed.  Updates to the list are automatically downloaded onto user's machine.  
+
+Email the following information to [ietioedge@microsoft.com][MailtoMicrosoftIetioedge] for your site to be added or removed from the IE compatibility list:  
+
+*   Owner name  
+*   Corporate title  
+*   Email address  
+*   Telephone number
```
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/web-platform/deprecating-ie-support.md]:
```md
*   Owner name  
*   Corporate title  
*   Email address  
*   Company name  
*   Street address  
*   Website address  
<!--  *   Telephone number  -->  
<!--  *   Target platform \(desktop, phone, Xbox\)  -->  
```

File: microsoft-edge/web-platform/deprecating-ie-support.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+Email the following information to [ietioedge@microsoft.com][MailtoMicrosoftIetioedge] for your site to be added or removed from the IE compatibility list:  
+
+*   Owner name  
+*   Corporate title  
+*   Email address  
+*   Telephone number  
+*   Company name  
+*   Street address  
+*   Website address  
+*   Target platform \(desktop, phone, Xbox\)
```
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/web-platform/deprecating-ie-support.md]:
```md
*   Owner name  
*   Corporate title  
*   Email address  
*   Company name  
*   Street address  
*   Website address  
<!--  *   Telephone number  -->  
<!--  *   Target platform \(desktop, phone, Xbox\)  -->  
```

File: microsoft-edge/web-platform/deprecating-ie-support.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+*   Email address  
+*   Telephone number  
+*   Company name  
+*   Street address  
+*   Website address  
+*   Target platform \(desktop, phone, Xbox\)  
+    
+<!-- links -->  
+
+[MailtoMicrosoftIetioedge]: mailto:ietioedge@microsoft.com "Send an email to ietioedge@microsoft.com"
```
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/web-platform/deprecating-ie-support.md]:
```md
[MailtoMicrosoftIetoedge]: mailto:ietoedge@microsoft.com "Send an email to ietoedge@microsoft.com"  
```

File: microsoft-edge/web-platform/deprecating-ie-support.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+## Why is a website added to the IE compatibility list?  
+
+The IE compatibility List only adds a website when the following actions occur.  
+
+*   Shows an IE user a message suggesting the user should use a different browser for compatibility reasons.  
+*   Owner requests to add the website to the IE compatibility list.  
+    
+## Update the IE compatibility list  
+
+The IE compatibility list is an XML file on [microsoft.com][MicrosoftOfficialHome].  The list is regularly updated in response to user's and website developers requests to have websites added or removed.  Updates to the list are automatically downloaded on user's machine.
```
Comment (Reezaali) [inline microsoft-edge/web-platform/deprecating-ie-support.md:29]:
```suggestion
The IE compatibility list is an XML file on [microsoft.com][MicrosoftOfficialHome].  The list is regularly updated in response to user's and website developers requests to have websites added or removed.  Updates to the list are automatically downloaded to user machines.  
```

**Review by Reezaali — APPROVED** [review]
Minor change suggested.

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**